modules/react-native-chat-v2/metadata.js

```diff
@@ -0,0 +1,13 @@
+const app_id = 'da4b3019-cfd7-430e-8153-d80dcef356e7'
+
+export default async (url) => {
+	try {
+		const res = await fetch(`https://opengraph.io/api/1.1/site/${encodeURIComponent(url)}?app_id=${app_id}`)
```

Comment:
We should consider a different approach here. This service is a paid service, that we would be hitting up on every message with a link (free plan has a limit of 100 monthly requests).